pkg/serial/handshake.go

```diff
@@ -32,10 +32,10 @@ import (
 )
 
 const (
-	SYN       = 0x16
-	ACK       = 0x06
-	DEBUG_ACK = 0x07
-	NAK       = 0x15
+	CodeSync     = 0x16
```

Comment:
Capital letter for constants and underscores. It actually was only about DEBUG_ACK. If you can think of a better name for DEBUG_ACK with no underscore I can roll changes back.